src/visualizers/Turtle.ts

```diff
@@ -126,8 +129,8 @@ class Turtle extends P5Visualizer(paramDesc) {
         super.setup()
         this.currentIndex = this.seq.first
         this.orientation = 0
-        this.X = this.sketch.width / 2 + this.start.x
-        this.Y = this.sketch.height / 2 + this.start.y
+        this.X = this.sketch.width / 2
+        this.Y = this.sketch.height / 2
 
```

Comment:
These changes to this.X and this.Y are a merge error, reverting a change from the prior PR. Please restore the lines as they are in ui2 now, thanks.